api/swagger.yaml

```diff
@@ -9700,6 +9700,7 @@ paths:
 
             Available filters:
             - `label` (`label=<key>`, `label=<key>=<value>`, `label!=<key>`, or `label!=<key>=<value>`) Prune volumes with (or without, in case `label!=...` is used) the specified labels.
+            - `all` (`all=true`) - Consider all (local) volumes for pruning and not just anonymous volumes.
```

Comment:
Looks like this PR was implemented to be included in API v1.42 (22.06); could you apply this change to the `docs/api/v1.42.yaml` file as well?